Update multi-factor-authentication-versions-plans.md

jboeshart opened the pull request:
Added clarification that the MFA for Administrators is specifically for Azure AD users with the Global Administrator role.

azurecla commented:
Hi __@jaymathe__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jaymathe). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

jboeshart committed:
Update multi-factor-authentication-versions-plans.md

Added clarification that the MFA for Administrators is specifically for Azure AD users with the Global Administrator role.

BuckeyeGuyJFlo commented on articles/multi-factor-authentication/multi-factor-authentication-versions-plans.md:
```diff
@@ -44,9 +44,9 @@ The following table provides a list of the features that are available in the va
+| Feature | Multi-Factor Authentication for Office 365 | Multi-Factor Authentication for Azure AD Administrators | Azure Multi-Factor Authentication |
 | --- |:---:|:---:|:---:|
-| Protect admin accounts with MFA |● |● (Global Administrator accounts only) |● |
+| Protect AAD admin accounts with MFA |● |● (AAD Global Administrator accounts only) |● |
```
We never are allowed to abbreviate Azure Active Directory as AAD it is Azure Active Directory or Azure AD

jboeshart committed:
Update multi-factor-authentication-versions-plans.md

Changed AAD references to Azure AD
